app/views/pages/dashboard.json.erb

```diff
@@ -0,0 +1,7 @@
+{
+"num_datasets": "<%= @datasets_count %>",
```

Comment:
We should render the integers as integers not strings. How about using https://github.com/rails/jbuilder?